lib/features/allow_ptrace.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * This feature adjusts the AppArmor profile to allow ptrace within a container.
+ * All of the interesting action takes place in `lib/task.js`.
+ */
+
+// Prefix used in scope matching for docker-worker features
+const FEATURE_SCOPE_PREFIX = 'docker-worker:feature:';
+
+export default class AllowPtrace {
+  constructor() {
+    this.featureName = 'allowPtrace';
+  }
+
+  async link(task) {
+    let docker = task.runtime.docker;
```

Comment:
Ok, cool.  Sorry, was just checking some things out while having coffee this morning.  I was just using this basic test: https://gist.github.com/gregarndt/ad2d538307c6d934813d

Also, for the tests to run, I think we need to mount /etc/apparmor.d into the container test/dockerworker.js uses.